spark/v3.2/spark/src/main/java/org/apache/iceberg/spark/procedures/RemoveOrphanFilesProcedure.java

```diff
@@ -121,6 +148,12 @@ public InternalRow[] call(InternalRow args) {
         action.compareToFileList(spark().table(fileListView));
       }
 
+      action.equalSchemes(equalSchemes);
+      action.equalAuthorities(equalAuthorities);
+      if (prefixMismatchMode != null) {
```

Comment:
nit: let's add an empty line before the if statement to match all other if statements above